frontend/packages/dev-console/src/components/NamespacedPage.tsx

```diff
@@ -1,12 +1,35 @@
 import * as React from 'react';
 import { NamespaceBar } from '@console/internal/components/namespace';
+import store from '@console/internal/redux';
+import { history } from '@console/internal/components/utils';
+import { createProjectModal } from '@console/internal/components/modals';
+import * as UIActions from '@console/internal/actions/ui';
 import ApplicationSelector from './dropdown/ApplicationSelector';
 
 import './NamespacedPage.scss';
 
+const CREATE_NEW_PROJECT = 'create-new-project';
+
+const onDropdownChange = (key: string) => {
+  if (key === CREATE_NEW_PROJECT) {
+    createProjectModal({
+      blocking: true,
+      onSubmit: (project) => history.push(`/add/ns/${project.metadata.name}`),
+    });
+  } else {
+    store.dispatch(UIActions.setActiveNamespace(key));
+  }
+};
+
 const NamespacedPage: React.FC = ({ children }) => (
   <div className="odc-namespaced-page">
-    <NamespaceBar>
+    <NamespaceBar
+      onChange={onDropdownChange}
+      actionItem={{
+        actionTitle: 'Create New Project',
+        actionKey: CREATE_NEW_PROJECT,
```

Comment:
Native k8s should have a `Create Namespace` action instead. You need to connect to flags to check if it's OpenShift or not.

Reply:
`namespace.jsx` uses `model.label`

Reply:
Fixed in 17abfd93855d2edac59d6e1e91676d53732aada4